README.md

```diff
@@ -25,7 +25,8 @@ use {
     "ThePrimeagen/refactoring.nvim",
     requires = {
         {"nvim-lua/plenary.nvim"},
-        {"nvim-treesitter/nvim-treesitter"}
+        {"nvim-treesitter/nvim-treesitter"},
+        {"nvim-lua/popup.nvim"}
```

Comment:
how is this a dependency?  Where do we use it at? (curious, because I don't know this)

Reply:
Oh prime, I think I switched things up... :(. I am. not sure about this anymore.

[telescope.nvim](https://github.com/nvim-telescope/telescope.nvim#installation) requires [nvim-lua/popup.nvim](https://github.com/nvim-lua/popup.nvim). But we don't have `telescope.nvim` as direct dependency. Garn.

However the example config in your [README.md](Readme.md),  makes use of `telescope.nvim` [here](https://github.com/ThePrimeagen/refactoring.nvim#setup). So may telescope is a requirement and therefore also `popup.nvim`?

At least installing `popup.nvim` solved #32 for me, but not sure if this is a general solution you'd want in the repo.

Reply:
Ahh I see.  Yes.  So I have telescope as a way to fuzzy find the refactoring, but not as a requirement itself.  

Perhaps a small comment above it and just make it telescope for telescope integration.